src/types.ts

```diff
@@ -125,6 +126,9 @@ export interface ForumPost {
 	likes: number
 	author: Player
 	postType: 0 | 1 | 2 | 3
+	anonymous: 0 | 1
```

Comment:
Like I said earlier, is there a reason to store this as 0 or 1 rather than boolean?